docs/_sass/_main.scss

```diff
@@ -182,6 +183,190 @@ button {
     align-content: center;
 }
 
+.logo {
+    width: 80%;
+}
+
+.search-icon {
+    margin: auto 0px;
+}
+
+.flex {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -moz-flex;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-flow: row wrap;
+    flex-flow: row wrap;
+    align-content: space-between;
+}
+
+#sidebar-search {
+    background-color: $color-green-appBG;
+    width: 400px;
+    height: 100vh;
+    position: fixed;
+    display: block;
+    top: 0;
+    right: 0;
+}
+
+@media only screen and (max-width: $breakpoint-tablet) {
+    #sidebar-search {
+        width: 100%;
+    }
+}
+
+.searchbar-title-wrapper {
+    padding: 20px;
+}
+
+.search-title {
+    font-size: 120%;
+}
+
+#toggle-search-close {
+    margin: auto;
+    margin-left: auto;
+    margin-right: 0px;
+}
+
+#sidebar-layer {
+    position: fixed;
+    /* Sit on top of the page content */
+    display: none;
+    /* Hidden by default */
+    width: 100%;
+    /* Full width (cover the whole page) */
+    height: 100%;
+    /* Full height (cover the whole page) */
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #1b574470;
```

Comment:
What is this color?

Reply:
This is for the background layer here @shawnborton 
<img width="1440" alt="Screenshot 2023-08-10 at 21 37 17" src="https://github.com/Expensify/App/assets/20473526/eb5d6b61-da14-4908-916d-b2a699eca5c2">

Reply:
This is changed to `background-color: rgba(0, 0, 0, 0.4);` as per NewDot design